### DISCUSSION TITLE
How to set a project reference to one platform of a MAUI class library from a console app

### DISCUSSION THREAD

**Discussion by UweReisewitz** [dotnet/maui#2950] [Q&A]
My MAUI project contains a MAUI class library. For the database I have a MigrationHelper App (.NET Console app). It needs the windows part of the MAUI class library.
Is it possible to reference that platform part directly or do I have to implement DI for this helper tool?

**Comment by KMWenyon** [comment]
any updates on this, I am trying to add an Android only project to a maui solution
